Sources/ILToJS/main.swift

```diff
@@ -0,0 +1,126 @@
+// Provides functionality in support of compiler development
```

Comment:
This will need the copyright header, you can just copy it from one of the other files.